astropy/time/tests/test_mask.py

```diff
@@ -158,7 +162,8 @@ def test_serialize_fits_masked(tmpdir):
     fn = str(tmpdir.join('tempfile.fits'))
     t = Table([tm])
     t.write(fn)
-    t2 = Table.read(fn, astropy_native=True)
+    with pytest.warns(VerifyWarning):
+        t2 = Table.read(fn, astropy_native=True)
```

Comment:
@taldcroft - is the VerifyWarning expected?

Reply:
Looks wrong to me, see #8773

Reply:
While having the warning is a bug, I think we should still catch it for now until the bug is fixed. So that when it is fix, this will cause the test to fail, and then we can remove the `pytest.warns` line?